.github/workflows/integration.yaml

```diff
@@ -0,0 +1,116 @@
+name: Integration
+
+on:
+  push:
+    branches:
+      - "main"
+  pull_request:
+    branches:
+      - "main"
+
+jobs:
+  integration:
+    name: Run integration tests
+    runs-on: ubuntu-latest
+    defaults:
+      run:
+        working-directory: ci/simple-case
+    steps:
+      - uses: actions/checkout@v3
+
+      - name: Run docker compose up
+        run: docker compose up -d
+
+      - name: Wait for service to be ready
+        run: sleep 10
```

Comment:
@piriwata Is it possible to use some APIs to check liveness for each component? I'd like to avoid use sleep because there are no guarantees whether 10 seconds is enough or not.

Reply:
Self note: we decided to implement more feasible solution later, so we use sleep for current tests.